File: arrays/sumofarray/sumofarray.java

import java.util.*;
import java.lang.*;
import java.io.*;

class GFG {
	public static void main (String[] args) {
		//code
		Scanner scan = new Scanner(System.in);
		int t = Integer.parseInt(scan.nextLine());
		for (int i = 0; i <t;i++) {
		    int s = Integer.parseInt(scan.nextLine());
		    String[] arrs = scan.nextLine().split(" ");
		    int[] arr = new int[s];
		    int  sum = 0 ;
		    for(int j =0; j <s; j++) {
		      sum += Integer.parseInt(arrs[j]);
		      //sum += scan.nextInt();(usw nextLine() whole program or use nextInt but dont we should not use both)
		    }
		    System.out.println(sum);
		    
		    
		}
	}
}

File: arrays/sumofarray/sumofarray.py
t = int(input())
for i in range(t):
    s = int(input())
    listt = list(map(int, input().split()))
    print(sum(listt))

# t = int(input());
# while t > 0:
#     t = t-1;
#     n = int(input());
#     arr = [int(i) for i in input().split()];
#     sum = 0;
#     for x in arr:
#         sum = sum + x;
#     print(sum);